plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -336,20 +381,22 @@ func TestGetRoundTripper(t *testing.T) {
 			server := httptest.NewServer(
 				http.HandlerFunc(
 					func(w http.ResponseWriter, r *http.Request) {
-						assert.Equal(t, "Bearer foo", r.Header.Get("Authorization"))
+						assert.Equal(t, "Bearer "+tc.wantBearer, r.Header.Get("Authorization"))
```

Comment:
This relates to the above suggestion:

```suggestion
						if tc.wantBearer != "" {
							assert.Equal(t, "Bearer "+tc.wantBearer, r.Header.Get("Authorization"))
						} else {
							assert.Empty(t, r.Header.Get("Authorization"))
						}
```